libraries/c_sdk/aws/defender/src/aws_iot_defender_v1.c

```diff
@@ -36,8 +36,15 @@
 /* Defender agent's status, initialized with eDefenderRepInit. */
 static DefenderReportStatus_t _status = eDefenderRepInit;
 
-static IotNetworkServerInfo_t _AWS_IOT_SERVER_INFO = AWS_IOT_NETWORK_SERVER_INFO_AFR_INITIALIZER;
-static IotNetworkCredentials_t _AWS_IOT_CREDENTIALS = AWS_IOT_NETWORK_CREDENTIALS_AFR_INITIALIZER;
+static IotNetworkServerInfo_t _serverInfo = AWS_IOT_NETWORK_SERVER_INFO_AFR_INITIALIZER;
```

Comment:
I believe ```aws_iot_defender_v1.c``` is not being used. Do we really need these changes?

Reply:
Yes, we need to investigate that. It was failing compilation and right now I don't want to modify cmake , project files , so just made changes so that it compiles.